bluesky/callbacks/broker.py

```diff
@@ -7,7 +7,34 @@
 import matplotlib.pyplot as plt
 
 
-class LiveImage(CallbackBase):
+class BrokerCallbackBase:
+    """
+    Base class for callbacks which need filled documents
+
+    Parameters
+    ----------
+    field: str
+        name of data field in an Event
+    fs: FileStore instance
+        The FileStore instance to pull the data from
+    """
+    def __init__(self, fields, *, fs=None):
+        if not isinstance(fields, (tuple, list)):
```

Comment:
This could be much more elegant I know that it has to be something we can loop through, but we can loop through strings now :frowning_face: